go/go.mod

```diff
@@ -60,6 +60,7 @@ require (
 	github.com/creasty/defaults v1.6.0
 	github.com/dolthub/go-mysql-server v0.14.1-0.20230329021806-7d5325285657
 	github.com/google/flatbuffers v2.0.6+incompatible
+	github.com/google/shlex v0.0.0-20191202100458-e7afc7fbc510
```

Comment:
FYI `This repository has been archived` https://github.com/google/shlex

Reply:
I think I'm fine with that, it's still publicly available and isn't going anywhere.